src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentEditor.js

```diff
@@ -9,9 +9,10 @@ import { DocumentForm } from './DocumentForm';
 export class DocumentEditor extends Component {
   constructor(props) {
     super(props);
+
     this.state = {
       data: {},
-      isLoading: true,
+      isLoading: !!props.match.params.documentPid,
```

Comment:
what is this solving? I can think only that there could be an issue if you render 2 forms in the same page...

Reply:
It doesn't solve anything actually, I purposefully added it for clarity :slightly_smiling_face: (for the form creation component although the state is not used, it was always in 'loading' which was confusing)

Reply:
if it is not used, maybe it's better to remove it then?

Reply:
@kprzerwa the state is used for edit forms but not for create since we do not need to fetch anything. It just feels wrong to have an incorrect value even though it is not used...

Reply:
I see, so it is used in the end :) I understood from the previous comment that it isn't